webview/src/experiments/components/table/TableHeaderCell.tsx

```diff
@@ -104,8 +104,7 @@ export const TableHeaderCell: React.FC<{
   setExpColumnNeedsShadow
 }) => {
   const [menuSuppressed, setMenuSuppressed] = React.useState<boolean>(false)
-  const isDraggable =
-    !column.placeholderOf && !['id', 'timestamp'].includes(column.id)
+  const isDraggable = !column.placeholderOf && !['id'].includes(column.id)
```

Comment:
Since there is only one item in `['id'].includes(column.id)` we can change it to be `column.id !== id`